src/Datadog.Trace/Agent/AgentWriter.cs

```diff
@@ -61,6 +61,11 @@ public void WriteTrace(Span[] trace)
             }
         }
 
+        public Task FlushAsync()
```

Comment:
Any reason you did just make `FlushTracesAsync()` public? The class is internal, so I don't think we need to worry about exposing it as a public API.

Reply:
Ok, sounds good, I'll use the FlushTracesAsync directly.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/802/commits/2b2ee99680880c3f41467916e08fecbb2deb93dd